bom/application/pom.xml

```diff
@@ -97,7 +97,7 @@
         <classmate.version>1.5.1</classmate.version>
         <!-- When updating, align bytebuddy.version to Hibernate needs as well (just below),
              as well as hibernate-orm.version-for-documentation in docs/pom.xml -->
-        <hibernate-orm.version>6.2.13.Final</hibernate-orm.version>
+        <hibernate-orm.version>6.2.18.Final</hibernate-orm.version>
         <bytebuddy.version>1.14.7</bytebuddy.version> <!-- Version controlled by Hibernate ORM's needs -->
         <hibernate-commons-annotations.version>6.0.6.Final</hibernate-commons-annotations.version> <!-- version controlled by Hibernate ORM -->
         <hibernate-reactive.version>2.0.6.Final</hibernate-reactive.version>
```

Comment:
Maybe upgrade to HR 2.0.7 while you're at it? It was supposedly the version to use with ORM 6.2.13.Final...

Reply:
Isn't https://github.com/hibernate/hibernate-reactive/commit/163eee7a4e9b6ee6b4ce183c735b43fbec795ae2 problematic as 3.6 branch is using Vert.x 4.4 - https://github.com/quarkusio/quarkus/blob/3.6/bom/application/pom.xml#L121

Reply:
@DavideD Any idea where this upgrade comes from? Seems strange to upgrade to a new vertx version in a micro of Hibernate Reactive...

Reply:
Sorry, which upgrade?  Hibernate Reactive uses Vert.x 4.4 since 2.0.0

Reply:
@DavideD The one @rsvoboda linked just above: https://github.com/hibernate/hibernate-reactive/commit/163eee7a4e9b6ee6b4ce183c735b43fbec795ae2
It's on the 2.0 branch, oddly, and also included in 2.0.7.Final.

Reply:
Probably a mistake on my part.